src/core/multifeaturelistmodel.cpp

```diff
@@ -47,6 +47,14 @@ void MultiFeatureListModel::setFeatures( QgsVectorLayer *vl )
   mSourceModel->setFeatures( requests );
 }
 
+void MultiFeatureListModel::setFeaturesForRectangle( QgsVectorLayer *vl, const QgsRectangle &rectangle )
```

Comment:
I would name it Extent instead of Reactangle? No strong opinion.

Reply:
@suricactus thanks i renamed it as you suggested